README.md

```diff
@@ -1,8 +1,74 @@
-# api-linter-service
+# API Linting Service
 
-[![SIT](https://img.shields.io/badge/SIT-awesome-blueviolet.svg)](https://jobs.schwarz)
```

Comment:
I think it would be better to keep the SIT Badge ;)

Reply:
done